code/game/jobs/job/religions.dm

```diff
@@ -0,0 +1,379 @@
+	name = "The Imperial Creed"
+	deity_name = "God-Emperor of Mankind"
+	bible_names = list("An Uplifting Primer", "Codex Astartes", "Codex Hereticus")
+	male_adept = "Confessor"
+	female_adept = "Prioress"
+	keys = list("imperium", "imperial cult")
+
+/datum/religion/toolboxia
+	name = "Toolboxia"
+	deity_name = "The Toolbox"
+	bible_name = "The Toolbox Manifesto"
+	male_adept = "Chief Assistant"
+	female_adept = "Chief Assistant"
+	keys = list("toolboxia", "toolbox")
+
+/datum/religion/homosexuality
+	name = "Homosexuality"
+	deity_name = "Steve Rambo" //Pushing Gaywards
+	bible_names = list("Guys Gone Wild", "Hunk Rump", "It's Okay to be Gay", "Daddy Gave You Good Advice")
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "LGBT Advocate"
+	female_adept = "LGBT Advocate"
+
+/datum/religion/homosexuality/misc(var/mob/living/carbon/human/H)
+	H.equip_or_collect(new /obj/item/clothing/under/darkholme(H), slot_w_uniform)
+
+/datum/religion/retard
+	name = "Retardation"
+	deity_name = "Brian Damag" //Ha
+	bible_names = list("Woody's Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition", "The Book of Pomf")
+	bible_type = /obj/item/weapon/storage/bible/booze
+	male_adept = "Retard"
+	female_adept = "Retard"
+	keys = list("lol", "wtf", "ass", "poo", "badmin", "shitmin", "deadmin", "nigger", "dickbutt", ":^)", "XD", "le", "meme", "memes", "ayy", "ayy lmao", "lmao", "reddit", "4chan", "tumblr", "9gag")
+
+/datum/religion/retard/misc(var/mob/living/carbon/human/H)
+	H.setBrainLoss(100) //Starts off retarded as fuck, that'll teach him
+
+/datum/religion/science
+	name = "Science"
+	deity_names = list("Albert Einstein", "Isaac Newton", "Niels Bohr", "Stephen Hawking")
+	bible_names = list("Principle of Relativity", "Quantum Enigma: Physics Encounters Consciousness", "Programming the Universe", "Quantum Physics and Theology", \
+							  "For I Have Tasted The Fruit", "Non-Linear Genetics", "The Mysteries of Bluespace", "Playing God: Collector's Edition")
+	male_adept = "Academician"
+	female_adept = "Academician"
+	keys = list("Science")
+
+/datum/religion/justice
+	name = "Justice"
+	deity_names = list("Almalexia", "Sotha Sil", "Vivec")
+	bible_name = "The 36 Lessons of Vivec"
+	male_adept = "Curate"
+	female_adept = "Curate"
+	keys = list("Justice", "Tribunal", "almsivi")
+
+/datum/religion/elder_scrolls
+	name = "Cult of the Divines"
+	deity_names = list("Talos", "Akatosh", "Dibella", "Stendarr", "Kynareth", "Mara", "Arkay", "Julianos", "Zenithar")
+	bible_name = "The Elder Scrolls"
+	male_adept = "Disciple of the Nine"
+	female_adept = "Disciple of the Nine"
+	keys = list("nine divines", "eight divines")
+
+/datum/religion/deadra
+	name = "Cult of the Deadreas"
+	deity_names = list("Azura", "Boethiah", "Sheogorath", "Sanguine", "Hircine", "Meridia", "Hermaeus Mora", "Nocturnal")
+	bible_name = list("The Blessings of Sheogorath", "Boethiah's Pillow Book", "Invocation of Azura")
```

Comment:
Probably out of scope, but Oghma Infinium could be a good alt-title for the bible?